build-all.sh

```diff
@@ -47,7 +47,10 @@ fi
 # Vulkan Renderer Lib
 if [ $PLATFORM = 'macos' ]
 then
-   VULKAN_SDK=/usr/local/
+   if [-z $VULKAN_SDK]
```

Comment:
Installation of VulkanSDK in system path is optional so the env variable is accounted for